src/reducers/profile/profileReducer.js

```diff
@@ -0,0 +1,45 @@
+import { profileActionTypes } from '../../actions/actionTypes';
+
+const initialState = {
+  profile: {},
+  isLoading: false,
+  editSuccess: false,
+};
```

Comment:
We already have a file initialState where we declare an initial state of a reducer from. Kindly shift this initialization to that file and import it wherever you need to use it.